datasets/thaiqa_squad/thaiqa_squad.py

```diff
@@ -0,0 +1,110 @@
+"""TODO(squad_v2): Add a description here."""
+
+from __future__ import absolute_import, division, print_function
+
+import json
+import os
+
+import datasets
+
+
+_CITATION = """\
+No clear citation guidelines from source:
+https://aiforthai.in.th/corpus.php
+SQuAD version:
+https://github.com/PyThaiNLP/thaiqa_squad
+"""
+
+_DESCRIPTION = """\
+`thaiqa_squad` is an open-domain, extractive question answering dataset (4,000 questions in `train` and 74 questions in `dev`) in
+[SQuAD](https://rajpurkar.github.io/SQuAD-explorer/) format, originally created by [NECTEC](https://www.nectec.or.th/en/) from
+Wikipedia articles and adapted to [SQuAD](https://rajpurkar.github.io/SQuAD-explorer/) format by [PyThaiNLP](https://github.com/PyThaiNLP/).
+"""
+
+
+class ThaiQaSquadConfig(datasets.BuilderConfig):
+    def __init__(self, **kwargs):
+        """BuilderConfig
+
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(ThaiQaSquadConfig, self).__init__(**kwargs)
+
+
+class ThaiQaSquad(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class ThaiqaSquad(datasets.GeneratorBasedBuilder):
```
This must be the camel case version of the dataset script name `thaiqa_squad`